low-level/files/opt/seagate/sspl/setup/sspl_setup.py

```diff
@@ -185,8 +182,8 @@ def validate(self):
         pass
 
     def process(self):
-        # TODO: Import relevant python script here for further execution.
-        pass
+        from cortx.sspl.lowlevel.files.opt.seagate.sspl.setup import sspl_setup_init
```

Comment:
Following way will look more cleaner, same was as in post_install
>from cortx.sspl.lowlevel.files.opt.seagate.sspl.setup.sspl_setup_init import SetupInit
SetupInit(self.args).process()